tests/testthat/test-tables.R

```diff
@@ -258,3 +258,76 @@ test_that("a correct but slightly pathological table correctly parsed", {
                c(rep('Categorical Columns', 2),
                  rep('Numerical Columns', 2)))
 })
+
+test_that("Test rowspans are respected when fill=True", {
+  select <- minimal_html("table parsing", '
```

Comment:
I wonder if it might be better to break this down into a few simpler tables that each illustrate one problem?